### DISCUSSION TITLE
Chat-GPT CSS style

### DISCUSSION THREAD

**Discussion by KirillRepinArt** [oobabooga/text-generation-webui#5004] [Show and tell]
[https://github.com/KirillRepinArt/ChatGPT-UI](url)

If anybody wants to use Chat-GPT like style, I've updated my style to be compatible with 20.12.2023 version of UI.

![TextGen_wip_22](https://github.com/oobabooga/text-generation-webui/assets/118350327/8a789210-f459-4b44-bba6-f4b9f5dbf501)
![TextGen_wip_20](https://github.com/oobabooga/text-generation-webui/assets/118350327/5f9b8b97-b6db-4564-845d-a562e90cff42)
![TextGen_wip_21](https://github.com/oobabooga/text-generation-webui/assets/118350327/acbf1a45-cdf7-4235-b7da-b83b22ef6ca9)
![TextGen_wip_19](https://github.com/oobabooga/text-generation-webui/assets/118350327/decb3846-9788-41da-bc19-7d04a3b10d62)

**Comment by KirillRepinArt** [comment]
I've update the style to work with the current version of UI (28th of April 2024):
https://github.com/KirillRepinArt/ChatGPT-UI.git
![TextGen_wip_27](https://github.com/oobabooga/text-generation-webui/assets/118350327/9386969f-21ec-4604-983d-b1ffafcfac4c)
![TextGen_wip_28](https://github.com/oobabooga/text-generation-webui/assets/118350327/d97fb561-4c3a-40b4-847b-54fd91d49276)
![TextGen_wip_29](https://github.com/oobabooga/text-generation-webui/assets/118350327/259ba708-d28c-4bcf-93ab-81fdcec5c0b7)

**Reply by thiagojramos** [reply]
![image](https://github.com/oobabooga/text-generation-webui/assets/45890502/777a484f-f1b3-4565-b9d8-449daa8abb52)

**Reply by KirillRepinArt** [reply]
My bad, updated the post.
